cocos/animation/marionette/graph-eval.ts

```diff
@@ -1968,6 +1999,56 @@ export class EmptyStateEval extends StateEval {
     }
 }
 
+class PoseStateEval extends StateEval {
+    public readonly kind = NodeKind.pose;
+
+    public elapsedTime = 0.0;
+
+    public readonly statusCache: MotionStateStatus = createStateStatusCache();
+
+    public constructor (state: PoseState, context: AnimationGraphBindingContext) {
+        super(state);
+        const poseEval = instantiatePoseGraph(state.graph);
```

Comment:
`poseEval` may be confused with `PoseStateEval`, How about 'poseGraphInstance' ?

Reply:
It's a pose node evaluation record, not pose graph itself. I've renamed it into `rootPoseNodeEval`: https://github.com/cocos/cocos-engine/pull/14779/commits/f68523da45c5b88cb4df9bd423012cde56d8d006